src/components/page-heading/template.njk

```diff
@@ -1,6 +1,6 @@
-<header class="hmrc-page-heading">
-  <h1 class="govuk-heading-xl">{{ params.text }}</h1>
+<header class="hmrc-page-heading{%- if params.classes %} {{ params.classes }}{%- endif -%}">
+  <h1 class="{{ params.headingClasses | default('govuk-heading-xl') }}">{{ params.text }}</h1>
   {%- if params.section -%}
-    <p class="govuk-caption-xl hmrc-caption-xl"><span class="govuk-visually-hidden">{% if params.language === 'cy' %}Teitl yr adran hon yw{%else%}This section is{%endif%} </span>{{ params.section }}</p>
+    <p class="{{ params.captionClasses | default('govuk-caption-xl hmrc-caption') }}"><span class="govuk-visually-hidden">{% if params.language === 'cy' %}Teitl yr adran hon yw{%else%}This section is{%endif%} </span>{{ params.section }}</p>
```

Comment:
I think the `.hmrc-caption` should be outside of the conditional as we always want it to display as a table-header-group.

Reply:
I have updated now, thanks for spotting that